src/components/commons/Dropdown/index.tsx

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import Image from 'next/image';
+
+import React, { useState } from 'react';
+
+import classNames from 'classnames/bind';
+import { useFormContext } from 'react-hook-form';
+
+import { SVGS } from '@/constants';
+
+import useTogglePopup from '@/hooks/useTogglePopup';
+
+import styles from './Dropdown.module.scss';
+
+const cx = classNames.bind(styles);
+
+type DropdownProps = {
+  name: string;
+  options: (number | string)[] | { id: number; title: string }[];
+  label?: string;
+  isSmall?: boolean;
+  isDisabled?: boolean;
+  setState?: (value: number) => void;
+};
+
+const Dropdown = ({
+  name,
+  label = '',
+  options,
+  isSmall = false,
+  isDisabled = false,
+  setState = undefined,
+}: DropdownProps) => {
+  const { register, setValue } = useFormContext();
+  const { isOpen, popupRef, buttonRef, togglePopup } = useTogglePopup();
```

Comment:
```suggestion
  const { isOpen, popupRef, buttonRef, togglePopup : handleInputClick  } = useTogglePopup();
```

   togglePopup을 구조분해해서 가져오는 부분에서 handleInputClick으로 함수명을 변경해서 사용하면 아래의 코드를 작성하지 않아도 될 거 같아요
   `const handleInputClick = () => {
    togglePopup();
  };`

Reply:
리뷰 남겨주신 것 보고 생각해보니 handleInputClick까지 없어도 될 것 같습니다!
handleInputClick 없이 그냥 togglePopup을 onClick에 전달하는 것으로 바꾸겠습니다!